docker-compose.yml

```diff
@@ -22,6 +22,12 @@ services:
       # We don't set up ssl certs for redis local dev
       REDIS_SSL_VALIDATION: none
       REDIS_URL: redis://db
+      AWS_REGION: us-east-1
+      AWS_KEY_ID: test
+      AWS_SECRET_KEY: test
+      AWS_BUCKET: ttbud-staging
```

Comment:
Should be ttbud-dev, and probably include a comment pointing to the shell script that sets this up